pkgs/applications/networking/instant-messengers/nheko/default.nix

```diff
@@ -43,15 +43,6 @@ mkDerivation rec {
     # https://github.com/NixOS/nixpkgs/pull/73940 are merged
```

Comment:
This was merged so it shouldn't be needed as well now.

Reply:
Yep, still builds fine without it.